conflux-sponsor-faucet-sdk/faucet/faucet.js

```diff
@@ -30,10 +36,18 @@ class Faucet {
     async tryTransact(call_func, params) {
         let nonce = Number(await this.cfx.getNextNonce(this.owner.address));
         let estimateData = await call_func(...params).estimateGasAndCollateral();
-        let gas = new BigNumber(estimateData.gasUsed)
+        let gas;
```

Comment:
I can see many duplicated codes in this repo for frontend and backend purpose.

Don't follow the way in Boomflow SDK, which is a bad code style.

Reply:
Basically, SDK helps users to interact with contract without ABI file. As for sign tx, it's up to js-conflux-sdk.

Reply:
rm backend part and leave sign-tx to js-conflux-sdk